app/models/concerns/feed_image_file.rb

```diff
@@ -0,0 +1,22 @@
+require 'active_support/concern'
+
+module FeedImageFile
+  extend ActiveSupport::Concern
+
+  included do
+    belongs_to :feed, touch: true
+    delegate :podcast, to: :feed, allow_nil: true
+  end
+
+  def destination_path
+    "#{feed.podcast.path}/#{feed_image_path}"
+  end
+
+  def published_url
+    "#{feed.podcast.base_published_url}/#{feed_image_path}"
+  end
+
+  def feed_image_path
+    "images/#{guid}/#{file_name}"
```

Comment:
NOTE: I opted to keep _all_ feed/itunes images in the current path scheme: `s3://prx-feed/<podcast>/images/<image_guid>/<filename>`.  So the feed slug never ends up in here, and our random image guids should never collide.  This keeps the `default_feed.feed_image` backwards compatible with the path of the previous `podcast.feed_image`.